Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.9-stretch AS builder
+WORKDIR /go/src/github.com/segmentio/aws-okta
+RUN useradd -u 10001 scratchuser
+RUN apt-get update && apt-get -y install libusb-dev  libusb-1.0-0-dev && pkg-config --cflags --libs libusb-1.0
+COPY . .
+RUN make container-build
+
+FROM debian:stretch-slim
+WORKDIR /app/
+RUN apt-get update && apt-get -y install libusb-1.0-0 ca-certificates python-pip && pip install awscli
```

Comment:
I wonder if we can statically link `libusb`? Then we'll have a standalone binary and can use a smaller image (both in terms of size and security footprint) like `alpine` or even `scratch`

Reply:
good gotcha let me try with CGO=0 during building time.

Reply:
I have trying different combinations I couldn't link libusb statically :S 

```
 CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags '-w -extldflags "-static"' -o mybin *.go
```
And it didn't work any clue here?

Reply:
Yeah, I don't have any pointers. If you can't get it to work, no worries. Nice to have